sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ExcludedPath.java

```diff
@@ -23,10 +23,11 @@ public ExcludedPath() {
     /**
      * Constructor.
      *
-     * @param jsonString the json string that represents the excluded path.
+     * @param path the excluded path.
      */
-    ExcludedPath(String jsonString) {
-        this.jsonSerializable = new JsonSerializable(jsonString);
+    public ExcludedPath(String path) {
```

Comment:
Now we have this constructor , what is the benefit of having default constructor and setPath() method explicitly .

Reply:
Good question, I am not so sure.

I think if path is required for includedPath, and we do not have any default path value, then should remove the default constructor.  (I guess same question for excludedPath).

@moderakh @kushagraThapar Any thoughts for this one?

Reply:
Good point, I say we remove default constructor, and just keep the constructor which takes `String path`

Reply:
what about setPath()? probably that shouldn't be public anymore now as path is set in the constructor.

Reply:
We can leave the `setPath()` API as it is, doesn't harm to have the API - and will be in sync with `setIndexes()` API.

Reply:
Removed default constructor from includedPath and excludedPath